packages/user-interface/src/pages/task/task-page.tsx

```diff
@@ -130,31 +134,37 @@ const TaskPage = () => {
   );
 
   return (
-    <React.Fragment>
-      <Helmet>
-        <link
-          rel="stylesheet"
-          href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
-        />
-        <link
-          rel="stylesheet"
-          href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
-        />
-        <link rel="stylesheet" href="https://cdn.form.io/formiojs/formio.full.min.css" />
-        <script src="https://cdn.form.io/formiojs/formio.full.min.js" />
-      </Helmet>
-      {!loading ? (
-        <Form
-          form={data?.getFormDefinitionById?.formDefinition}
-          formReady={redrawForm}
-          submission={submission}
-          onChange={setFormSubmission}
-          onSubmit={onFormSubmit}
-        />
-      ) : (
-        getSubmittedMessage()
-      )}
-    </React.Fragment>
+      <section className={styles.case}>
+        {!error ? (
+          <React.Fragment>
+            <Helmet>
+              <link
+                rel="stylesheet"
+                href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+              />
+              <link
+                rel="stylesheet"
+                href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+              />
+              <link rel="stylesheet" href="https://cdn.form.io/formiojs/formio.full.min.css" />
+              <script src="https://cdn.form.io/formiojs/formio.full.min.js" />
+            </Helmet>
+            {!loading ? (
+              <Form
+                form={data?.getFormDefinitionById?.formDefinition}
+                formReady={redrawForm}
+                submission={submission}
+                onChange={setFormSubmission}
+                onSubmit={onFormSubmit}
+              />
+            ) : (
+              getSubmittedMessage()
+            )}
+          </React.Fragment> ) : (<Paragraph>
```

Comment:
Kun je dit als een losse return terug geven als de error true is: 

```suggestion
          if (error) return (
          <Paragraph>...</Paragraph>
          )
```